### PR TITLE
OSDOCS-8923: adds RHEL 9.3 to MicroShift V4.15 relnotes

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.15 only

Issue:
[OSDOCS-8923](https://issues.redhat.com/browse/OSDOCS-8923)

Link to docs preview:
[Release notes RHEL](https://69919--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes#microshift-4-15-rhel)

QE review:
- N/A

### CODE REVIEW

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -30,8 +30,8 @@ This release adds improvements related to the following components and concepts.
 
 //L3 major categories with features in each as L4s, for example:
 [id="microshift-4-15-rhel"]
-=== {op-system-base-full} {op-system-version}
-* {microshift-short} runs on {op-system-base-full} versions 9.2 and 9.3.
+=== {op-system-base-full}
+* {microshift-short} now runs on {op-system-base-full} versions 9.2 and 9.3. Compatibility with {op-system} is an enhancement for {microshift-short} version 4.15
```
Comment (DanielFroehlich) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:34]: It dont understand the "compatibility...." sentence. V4.14 also has been compatible with RHEL.

**Review by DanielFroehlich — CHANGES_REQUESTED** [review]
Thats only the release notes.

- RHEL Versions are also mention in the system requirements (https://69919--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm#microshift-install-system-requirements_microshift-install-rpm) 

That needs an update. We might want to change this to a generic "A supported version of RHEL, see the compatbility matrix"  in the RHDE docs, to avoid needing to adjust it the next time this changes (expected with V4.16).

File: microshift_release_notes/microshift-4-15-release-notes.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-15-release-notes.adoc]: Supposed to say "...with 9.3"; added.

**Review by DanielFroehlich — APPROVED** [review]

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 2]
```diff
@@ -30,8 +30,8 @@ This release adds improvements related to the following components and concepts.
 
 //L3 major categories with features in each as L4s, for example:
 [id="microshift-4-15-rhel"]
-=== {op-system-base-full} {op-system-version}
-* {microshift-short} runs on {op-system-base-full} versions 9.2 and 9.3.
+=== {op-system-base-full}
+* {microshift-short} now runs on {op-system-base-full} versions 9.2 and 9.3. Compatibility with {op-system} 9.3 is an enhancement for {microshift-short} version 4.15
```
Comment (skopacz1) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:34]: I'd just add a period at the end of the second sentence.
```suggestion
* {microshift-short} now runs on {op-system-base-full} versions 9.2 and 9.3. Compatibility with {op-system} 9.3 is an enhancement for {microshift-short} version 4.15.
```

**Review by skopacz1 — COMMENTED** [review]
One nit but otherwise LGTM!

**Pull request merged**